### PR TITLE
chore: remove ecommerce plugin as it is deprecated

### PR DESCRIPTION
**Pull request by Faraz32123**
- remove ecommerce plugin from nightly as it is deprecated.
- check tutor-ecommerce deprecation issue for more info: overhangio/tutor-ecommerce#83

### CODE REVIEW

**Comment by Faraz32123** [comment]
tutor-ecommerce was already removed from main [here](https://github.com/overhangio/tutor/pull/1180/commits/fbfef07339c0cf5b01902f953ef7ab91feab1075).

Made this PR to remove ecommerce completely from tutor main(nighlty).
@regisb @DawoudSheraz need your review on this.

File: tutor/plugins/v0.py [hunk 1]
```diff
@@ -282,7 +282,6 @@ class OfficialPlugin(BasePlugin):
     NAMES = [
         "android",
         "discovery",
-        "ecommerce",
```
Comment (DawoudSheraz) [inline tutor/plugins/v0.py:285]: not sure if this should be removed from v0 as it can serve some backwards compatibility purposes.

Comment (Faraz32123) [reply tutor/plugins/v0.py:285]: yes, this needs to be discussed...

Comment (regisb) [reply tutor/plugins/v0.py:285]: This class is only meant for the discovery of plugins in the official github-built binary (see `make bundle`). We don't want the ecommerce plugin there, at least not in the main branch, so we should remove it from this list.

**Review by DawoudSheraz — APPROVED** [review]

**Review by regisb — APPROVED** [review]
lgtm!

**Pull request merged**